.buildkite/publish/publish-common.sh

```diff
@@ -13,9 +13,6 @@ export SCRIPT_DIR="$CURDIR"
 export BUILDKITE_DIR=$(realpath "$(dirname "$SCRIPT_DIR")")
 export PROJECT_ROOT=$(realpath "$(dirname "$BUILDKITE_DIR")")
 
-# temporary hard-coded build qualifier, this should be removed once we've released 9.0
-export VERSION_QUALIFIER="${VERSION_QUALIFIER:-beta1}"
```

Comment:
Removing this on `main`, since I've created a 9.0 branch where this should be for now